apps/src/templates/lessonOverview/activities/LevelDetailsDialog.jsx

```diff
@@ -74,7 +83,41 @@ export default class LevelDetailsDialog extends Component {
         </div>
       );
     } else {
-      return <div>Not implemented yet</div>;
+      // TODO: calculate more of these parameters based on the level and pages
+      return (
+        <UnconnectedTopInstructions
+          hasContainedLevels={false}
+          noVisualization={true}
+          isMinecraft={false}
+          isBlockly={false}
+          isRtl={false}
+          longInstructions={level.longInstructions || level.long_instructions}
+          shortInstructions={level.shortInstructions}
+          noInstructionsWhenCollapsed={true}
+          levelVideos={level.videos}
+          mapReference={level.mapReference}
+          referenceLinks={level.referenceLinks}
+          teacherMarkdown={level.teacherMarkdown}
+          viewAs={this.props.viewAs}
+          height={this.state.height}
+          maxHeight={Math.min(this.state.maxHeight, MAX_LEVEL_HEIGHT)}
```

Comment:
Is there a reason for the dialog to not allow the instructions to expand taller than 400?

Reply:
Mostly chosen arbitrarily, though it seems like it would be good to have a max. Definitely open to suggestions on this or a way to make this scale to window size a bit better.

Reply:
I'm totally on board with a max height. Do dialogs have a max height already based on the size of the window? If some I would say for now we should use as much of that as we can if the instructions are long and need it. Preventing scrolling if possible is my main concern

Reply:
Oh good point, looks like dialogs have a max height of 80% of the window. The problem is that it creates some double scrollbar issues because both the instructions and the dialog will add scrollbars. I think this is because TopInstructions does some fancy logic to figure out how much space each component has and I'm having trouble figuring out how to get that max height to TopInstructions.

Reply:
Whats the min screen size height we support? Maybe we pick something just slightly smaller than 80% of that so you don't get the double scroll?

Reply:
From https://support.code.org/hc/en-us/articles/202591743, min height is 728 px, 80% of that is 582. So I could choose 500 or 550 px, to give a bit of room for the header and buttons.

Reply:
I think TopInstructions does a bit of logic that I don't fully understand but I upped that max to 550. It doesn't always use that max space but that's because of something in top instructions that I don't fully understand.


No double scroll though until about 600px in height! 

https://user-images.githubusercontent.com/46464143/110866036-614caa00-8279-11eb-8596-e802f7d938bc.mov

Reply:
As I recall, TopInstructions does a _lot_ of magic to get all its various height requirements to work.

Reply:
Yeah, I thought I understood some of them but I guess less than I though 🙃